### PR TITLE
Add `<For/>` component

### PR DESCRIPTION
**Pull request by hyanmandian**
Like my previous PR #3 , I was inspired by https://github.com/AlexGilleran/jsx-control-statements and I created the  `<For />` component too.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -109,6 +109,28 @@ Or you could just use plain JavaScript:
 </div>
 ```
 
+### `<For>`
+
+React component that iterates over `of` prop and renders the children.
```
Comment (sindresorhus) [inline readme.md]: This is not correct. It doesn't render the `children`, but rather the result the `render` prop.

File: index.js [hunk 2]
```diff
@@ -32,6 +32,13 @@ If.propTypes = {
 	render: PropTypes.func
 };
 
+export const For = ({render, children, of}) => of.map((item, index) => render ? render(item, index) : children(item, index));
```
Comment (sindresorhus) [inline index.js]: I think we should pick on way to do it. Let's go with the `render` variant and drop the `children` variant.

**Comment by sindresorhus** [comment]
Can you fix the merge conflicts?

File: index.js (hunk 2 above)
Comment (hyanmandian) [reply index.js]: Good point, I will change it

**Pull request merged**